test/IRGen/package_resilience.swift

```diff
@@ -214,29 +226,43 @@ package class ClassWithResilientThenEmpty {
 
 // CHECK-LABEL: define{{.*}} swiftcc {{i32|i64}} @"$s18package_resilience38memoryLayoutDotSizeWithResilientStructSiyF"()
 public func memoryLayoutDotSizeWithResilientStruct() -> Int {
-  // CHECK: ret [[INT]] [[#WORDSIZE + WORDSIZE]]
+// CHECK: [[FIELD_VALUE:%.*]] = load ptr, ptr {{.*}},
+// CHECK-NEXT: [[FIELD_PTR:%.*]] = getelementptr inbounds %swift.vwtable, ptr [[FIELD_VALUE]],
+// CHECK-NEXT: [[SIZE:%.*]] = load [[INT]], ptr [[FIELD_PTR]],
+// CHECK-NEXT: ret [[INT]] [[SIZE]]
   return MemoryLayout<Size>.size
 }
 
 // CHECK-LABEL: define{{.*}} swiftcc {{i32|i64}} @"$s18package_resilience40memoryLayoutDotStrideWithResilientStructSiyF"()
 public func memoryLayoutDotStrideWithResilientStruct() -> Int {
-  // CHECK: ret [[INT]] [[#WORDSIZE + WORDSIZE]]
+// CHECK: [[FIELD_VALUE:%.*]] = load ptr, ptr {{.*}},
+// CHECK-NEXT: [[FIELD_PTR:%.*]] = getelementptr inbounds %swift.vwtable, ptr [[FIELD_VALUE]],
+// CHECK-NEXT: [[SIZE:%.*]] = load [[INT]], ptr [[FIELD_PTR]],
+// CHECK-NEXT: ret [[INT]] [[SIZE]]
```

Comment:
These paths have regressed

Reply:
If I return MemoryLayout<PublicSize>.size, where PublicSize is defined as:
```
public struct PublicSize {
  public var w: Int
  public let h: Int 

  public init(w: Int, h: Int) {
    self.w = w
    self.h = h
  }

  public func method() {}
  public mutating func mutantMethod() {}
}
```
I get the following result with and without this PR.
```
define swiftcc i64 @"$s18package_resilience46memoryLayoutDotStrideWithPublicResilientStructSiyF"() #0 {
entry:
  %0 = call swiftcc %swift.metadata_response @"$s16resilient_struct10PublicSizeVMa"(i64 0) #14
  %1 = extractvalue %swift.metadata_response %0, 0
  %2 = getelementptr inbounds ptr, ptr %1, i64 -1
  %.valueWitnesses = load ptr, ptr %2, align 8, !invariant.load !23, !dereferenceable !24
  %3 = getelementptr inbounds %swift.vwtable, ptr %.valueWitnesses, i32 0, i32 8
  %size = load i64, ptr %3, align 8, !invariant.load !23
  ret i64 %size
}
```

This PR is to make the `package` just as resilient as non-frozen `public`. Optimization PR to make it non-resilient will follow later.